pkgs/by-name/ro/ropr/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "ropr";
+  version = "0.2.26";
+
+  src = fetchFromGitHub {
+    owner = "Ben-Lichtman";
+    repo = "ropr";
+    rev = "ec3eb2d91d9b4a940a8013a079ead47d7eab6dac";
+    hash = "sha256-iN6CSivyBe6Ibbl+oQ2wThbSyHTKne14XsilkMntnfE=";
+  };
+
+  cargoHash = "sha256-4YEriANTAt1dx9bXhlHFN+kNLde+8BLocuhXdFG24xo=";
+
+  meta = {
+    description = "Multithreaded ROP gadget finder for x86(_64)";
+    homepage = "https://github.com/Ben-Lichtman/ropr";
+    license = lib.licenses.mit; # dual MIT and asl20
```

You can use lists too:
```suggestion
    license = with lib.licenses; [
      mit
      asl20
    ];
```

---

TIL, thanks for the suggestion.